torch/_tensor_docs.py

```diff
@@ -4102,6 +4102,75 @@ def callable(a, b) -> number
     >>> torch.equal(b, c)
     False
 
+
+view(dtype) -> Tensor
+
+Returns a new tensor with the same data as the :attr:`self` tensor but of a
+different :attr:`dtype`.
+
+The returned tensor shares the same data and must have the same number
+of elements, but may have a different dtype. For a tensor to be viewed, the new
+dtype must have the same number of bytes with its original dtype.
+
+.. warning::
```

Comment:
Change the warning to a note and change the note to be just 

"This overload is not supported by TorchScript, and using it in a Torchscript program will cause undefined behavior."

Reply:
Could we blacklist this overload in torchscript to avoid this kind of confusing error?

Reply:
@albanD This overload is already blacklisted in JIT. But unfortunately, this is not enough. What you write is
```
x.view(torch.int8)
```
but what JIT see is
```
x.view(1)
```
and there is no easy way for JIT to know if you wrote `view(1)` or `view(torch.int8)`